tests/Command/RemoveThumbsCommandTest.php

```diff
@@ -66,22 +66,15 @@ protected function setUp(): void
     {
         parent::setUp();
 
-        $this->container = new Container();
+        $this->mediaManager = $this->createMock(MediaManagerInterface::class);
+        $this->pool = $this->createMock(Pool::class);
```

Comment:
```suggestion
        $this->pool = $this->createStub(Pool::class);
```

Reply:
What is diffrence between it?

Reply:
The difference is that, unlike mocks, stubs are not intended to perform expectations. Please, see https://github.com/sebastianbergmann/phpunit-mock-objects/issues/411.